src/Components/Search/NewSearch/NewSearchBarInput.tsx

```diff
@@ -1,70 +1,184 @@
+import { useTracking } from "react-tracking"
+import * as DeprecatedSchema from "@artsy/cohesion/dist/DeprecatedSchema"
+import { getENV } from "Utils/getENV"
+// eslint-disable-next-line no-restricted-imports
+import request from "superagent"
+import { StaticSearchContainer } from "./StaticSearchContainer"
+// import { DESKTOP_NAV_BAR_TOP_TIER_HEIGHT } from "Components/NavBar/constants"
 
 const logger = createLogger("Components/Search/NewSearchBar")
+const SEARCH_THROTTLE_DELAY = 500
 
 export interface NewSearchBarInputProps extends SystemContextProps {
   relay: RelayRefetchProp
   viewer: NewSearchBarInput_viewer$data
   isXs: boolean
 }
 
+const reportPerformanceMeasurement = performanceStart => {
+  const duration = performance.now() - performanceStart
+  const deviceType = getENV("IS_MOBILE") ? "mobile" : "desktop"
+
+  const metricPayload = {
+    name: "autocomplete-search-response",
+    tags: [`device-type:${deviceType}`, "design:rich"],
+    timing: duration,
+    type: "timing",
+  }
+
+  request
+    .post(getENV("VOLLEY_ENDPOINT"))
+    .send({
+      metrics: [metricPayload],
+      serviceName: "force",
+    })
+    .end()
+}
+
 const NewSearchBarInput: FC<NewSearchBarInputProps> = ({
   isXs,
   relay,
   viewer,
 }) => {
+  const tracking = useTracking()
   const { t } = useTranslation()
-  const [value, setValue] = useState("")
+  const [value, setValue] = useState(getSearchTerm(window.location))
+  const [selectedPill, setSelectedPill] = useState<PillType>(TOP_PILL)
+  const [searchIncrement, setSearchIncrement] = useState(0)
 
   const options = extractNodes(viewer.searchConnection)
-  const formattedOptions = options.map(option => {
-    return {
-      text: option.displayLabel!,
-      value: option.displayLabel!,
-      subtitle: getLabel({
-        displayType: option.displayType ?? "",
+  const formattedOptions: SuggionItemOptionProps[] = options.map(
+    (option, index) => {
+      return {
+        text: option.displayLabel!,
+        value: option.displayLabel!,
+        subtitle:
+          getLabel({
+            displayType: option.displayType ?? "",
+            typename: option.__typename,
+          }) ?? "",
+        imageUrl: option.imageUrl!,
+        showArtworksButton: !!option.statuses?.artworks,
+        showAuctionResultsButton: !!option.statuses?.auctionLots,
+        href: option.href!,
         typename: option.__typename,
-      }),
-      imageUrl: option.imageUrl!,
-      showArtworksButton: !!option.statuses?.artworks,
-      showAuctionResultsButton: !!option.statuses?.auctionLots,
-      href: option.href!,
+        item_number: index,
+        item_type: option.displayType!,
+      }
     }
-  })
+  )
 
-  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
-    setValue(event.target.value)
+  useUpdateEffect(() => {
+    tracking.trackEvent({
+      action_type:
+        options.length > 0
+          ? DeprecatedSchema.ActionType.SearchedAutosuggestWithResults
```

Comment:
Ditto (here and elsewhere) - if these events aren't actually deprecated, could y'all open a PR and copy them into the canonical (current) analytics schema?

Reply:
Thanks! Same as here: https://github.com/artsy/force/pull/12308#discussion_r1185907851